Source/PrivateHeaders/Ably/ARTRealtimePresence+Private.h

```diff
@@ -1,9 +1,12 @@
+#import <Ably/ARTPresenceMap.h>
 #import <Ably/ARTRealtimePresence.h>
 #import <Ably/ARTRealtimeChannel+Private.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface ARTRealtimePresenceInternal : NSObject <ARTRealtimePresenceProtocol>
+@interface ARTRealtimePresenceInternal : NSObject <ARTRealtimePresenceProtocol, ARTPresenceMapDelegate>
+
+@property (readonly, nonatomic) ARTEventEmitter<ARTEvent *, ARTPresenceMessage *> *presenceEventEmitter;
```

Comment:
This change seems to be in the wrong commit.

Reply:
646f555a1e037aaa2a43ebe440343e3ff1396075